cypress/e2e/po/pages/explorer/custom-resource-definitions.po.ts

```diff
@@ -0,0 +1,52 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import BurgerMenuPo from '@/cypress/e2e/po/side-bars/burger-side-menu.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import CodeMirrorPo from '@/cypress/e2e/po/components/code-mirror.po';
+import CustomResourceDefinitionsCreateEditPo from '@/cypress/e2e/po/edit/custom-resource-definitions.po';
+
+export class CustomResourceDefinitionsPagePo extends PagePo {
```

Comment:
Comment only. There's a lot of boilerplate / c&p now for resource list pages.

At some point in the future we should create an abstract class ResourceList that these type of pages can extend, then override specific things like createPath, navTo, etc